lib/blobs/usbboot/cmdline.txt

```diff
@@ -0,0 +1 @@
+console=serial0,115200
```

Comment:
This is only useful for debugging, so should be removed before we actually make the next production version of Etcher.

Reply:
Should I keep an empty `cmdline.txt`, or remove the file altogether?

Reply:
usb-boot seems to happily skip over any files that it can't find. So should be safe to remove it.

Reply:
Done!